.cirrus.yml

```diff
@@ -1,5 +1,5 @@
 container:
-  image: gradle:jdk17
+  image: gradle:jdk8
```

Comment:
Yes, I think it would be more clean this way, the previous setup produced some weird issues around Javadoc for example.

Reply:
Could you, please, tell more about these weird issues around Javadoc? I didn't understand what you are talking about.

Reply:
I have seen messages like these:
```
> Task :javadoc
/home/esta/IdeaProjects/github/openweathermap-java-api/src/main/java/com/github/prominence/openweathermap/api/mapper/AbstractMapper.java:37: warning: no comment
public abstract class AbstractMapper {
                ^
/home/esta/IdeaProjects/github/openweathermap-java-api/src/main/java/com/github/prominence/openweathermap/api/mapper/AbstractMapper.java:38: warning: no comment
    protected final ObjectMapper objectMapper = new ObjectMapper();
                                 ^
/home/esta/IdeaProjects/github/openweathermap-java-api/src/main/java/com/github/prominence/openweathermap/api/mapper/AbstractMapper.java:40: warning: no comment
    protected LocalDateTime parseDateTime(JsonNode dateTimeNode) {
                            ^
```

The ones above are from the `dev` branch I started my own branch from. The strange thing was, for example when the Lombok generated methods were reported too, despite the fact that normally they would be marked as generated.

Also, another thing I have noticed. When I tried to actually use the library on Java 8 after I started to rewrite it, Java was complaining about `List.copyOf()` and similar newer collection API methods (I suspect because the class exists on both versions, but since the method is new, the compiler messed up something). Not sure whether it was just my bad luck or something, because as I mentioned, I usually prefer the convention over configuration approach, using JDK8 purely if the library should support JDK8 to eliminate any possible errors.

I hope it makes sense!

Reply:
Not sure whether I had the same problems but fine, let's stay of JDK8 for now.